installer/dev/main.cpp

```diff
@@ -6,6 +6,11 @@
 #include "install.h"
 #include "resource.h"
 
+#ifndef GUID_NULL
+struct __declspec(uuid("00000000-0000-0000-0000-000000000000")) GUID_NULL;
+#define GUID_NULL __uuidof(struct GUID_NULL)
+#endif
```

Comment:
Why not

```
constexpr GUID GUID_NULL{};
```
?

Reply:
no good reason - lifted the pattern from OS code